### ISSUE TITLE
find_package(args REQUIRED) doesn't set args_VERSION

### ISSUE THREAD

**Issue by yurivict** [Taywee/args#109]
It's common for cmake-discovered packages to set the ```{name}_VERSION``` variable.

**Comment by Taywee** [comment]
Yeah, that's definitely an issue here.  When packaging and pulling in args, I get these variables:

```
-- args_FOUND=1
-- args_VERSION_COUNT=0
-- args_VERSION_MAJOR=0
-- args_VERSION_MINOR=0
-- args_VERSION_PATCH=0
-- args_VERSION_TWEAK=0
```

This is supposed to be set via `find_package`.  Honestly, I'm not exactly sure how I'm supposed to fix this.  CMake is not fun to work with.  I'll do some digging and see if I can find something, but I'd be more than happy to accept a pull request that addresses it too.

**Comment by Taywee** [comment]
Figured it out.  This is solved in version 6.4.3.

```
-- args_FOUND=1
-- args_VERSION=6.4.3
-- args_VERSION_COUNT=3
-- args_VERSION_MAJOR=6
-- args_VERSION_MINOR=4
-- args_VERSION_PATCH=3
-- args_VERSION_TWEAK=0
```

**Comment by Taywee** [comment]
The version is now released.  Thank you for the report.  I'm certain the CMakeLists.txt still has tons of problems, but it's hard to be motivated to work on it when it's still CMake in the end.
